confidential-data-hub/docs/SEALED_SECRET.md

```diff
@@ -140,25 +173,30 @@ Then add a prefix `sealed.`
 sealed.ewoJInZlcnNpb24iOiAiMC4xLjAiLAoJInR5cGUiOiAiZW52ZWxvcGUiLAoJInByb3ZpZGVyIjogInh4eCIsCgkia2V5X2lkIjogInh4eCIsCgkiZW5jcnlwdGVkX2tleSI6ICJhYjI3ZGM9IiwgCgkiZW5jcnlwdGVkX2RhdGEiOiAieHh4IiwKCSJ3cmFwX3R5cGUiOiAiQTI1NkdDTSIsCgkiaXYiOiAieHh4IiwKCSJhbm5vdGF0aW9ucyI6IHsKCQkiY3J5cHRvX2NvbnRleHQiOiB7CgkJCSJhbGdvcml0aG0iOiAiQTI1NkdDTSIKCQl9LAoJCSJwcm92aWRlcl9zZXR0aW5nIjogewoJCQkia21zX2luc3RhbmNlX2lkIjogInh4eCIKCQl9Cgl9Cn0=
```

Comment:
I am  afraid that we need also to update this document part. Sealed secret format should be like `"sealed".<JWS header>.<JWS body, s.t. base64 payload>.<signature>`. But for now, `<JWS header>` and `<signature>` are ignored. See [example](https://github.com/kata-containers/kata-containers/pull/9719/files#diff-8bc9cea233848b1c184a4052fee2b24d8124539de2aa25c9efa3458b298ccb03R63)

Reply:
Ah right. I updated the docs.

Btw do you think this secret-cli tool should output a json secret or the base64 format?

Reply:
The final aim is to output the format `"sealed".<JWS header>.<JWS body, s.t. base64 payload>.<signature>`. But currently, I am ok with both.